src/cmodel.c

```diff
@@ -1427,6 +1429,8 @@ cmodel_t *CM_LoadMap (char *name, qbool clientload, unsigned *checksum, unsigned
 
 	strlcpy (map_name, name, sizeof(map_name));
 
+	Hunk_TempFlush();
```

Comment:
not obvious, an explanation is required.

Reply:
It could be left out, especially as there should be a lot of hunk space available now compared to before this PR. It's almost free, what might it be, microseconds for the memset? Idea was to not leave waste after load, and let mods allocate memory to their heart's desire without artificial bounds.

Reply:
I understand why you did it, a brief explanation would save readers time.